### ISSUE TITLE
Crash while placing tin item pipes (Possibly Unrelated)

### ISSUE THREAD

**Issue by ToastedEggplant** [GTNewHorizons/GT-New-Horizons-Modpack#4980]
#### Which modpack version are you using?
Latest
#
#### If in multiplayer; On which server does this happen? 
Private (no modifications)
#
#### What did you try to do, and what did you expect to happen?
I was placing tin item pipes
#
#### What happened instead? (Attach screenshots if needed)
Server closed and crashed with this log https://pastebin.com/ePD0e0b7
#
#### What do you suggest instead/what changes do you propose?
I'm not sure. I'm not sure if this was even related to the pipes nor do I have any idea what this was related to. Hopefully someone with better understanding can figure it out.

**UPDATE EDIT**

So it happened again placing lead fluid pipes this time https://pastebin.com/S5evdQp1

**Comment by Dream-Master** [comment]
![image](https://user-images.githubusercontent.com/2970688/64375967-706cef00-d027-11e9-8741-634509238556.png)
placing both get no issue

**Comment by mitchej123** [comment]
```
java.util.ConcurrentModificationException
    at java.util.ArrayList$Itr.checkForComodification(ArrayList.java:909)
    at java.util.ArrayList$Itr.next(ArrayList.java:859)
    at net.minecraft.world.World.func_72939_s(World.java:1933)  # UpdateEntities
    at net.minecraft.world.WorldServer.func_72939_s(WorldServer.java:489) # UpdateEntities
    at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:636) # UpdateTimeLightsAndEntities
    at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:334)
    at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:547)
    at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:427)
    at net.minecraft.server.MinecraftServer$2.run(MinecraftServer.java:685)
```

This doesn't look GT5U pipe related at all.  Weird.  Corrupt chunk maybe?

**Comment by ljjmrlion** [comment]
The server was lagging a little when I placed the pipes, they hadn't fully rendered the connections yet.
The crashes did not consistently occur when placing the pipes, only when the server lagged a tiny bit and the pipe didn't render the connections quickly. Each pipe I placed at the time of the crash also happened to be in seperate chunks. The pipes in each crash are also not connected to eachother.
